kuksa_databroker/databroker-cli/src/cli.rs

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
-* Copyright (c) 2022 Contributors to the Eclipse Foundation
+* Copyright (c) 2023 Contributors to the Eclipse Foundation
```

Comment:
Why changing year? Eclipse seems to recommend either just stating the year the file was created, or a range "creation year - year of latest change", see https://www.eclipse.org/projects/handbook/#ip-copyright-headers